content/providers/list/2.hardware.yaml

```diff
@@ -4,37 +4,40 @@ id: hardware
 providers:
 - codeExamples:
   - fullCode: |-
+      from qiskit import QuantumCircuit
+
       from qiskit_aqt_provider import AQTProvider
-      aqt = AQTProvider('MY_TOKEN')
-      backend = aqt.get_backend('aqt_innsbruck')
+      from qiskit_aqt_provider.primitives import AQTSampler
 
-      # Build circuit
-      from qiskit import QuantumCircuit
-      circuit = QuantumCircuit(2, 2)
+      # Define a circuit
+      circuit = QuantumCircuit(2)
       circuit.h(0)
-      circuit.cx(0,1)
-      circuit.measure([0,1], [0,1])
+      circuit.cnot(0, 1)
+      circuit.measure_all()
```

Comment:
The new code and the old code is equivalent. Do you really need to update it? This is purely Qiskit (not the provider) so I would prefer to have the same code across different provider.

Reply:
Sure. I reverted the change.